dpnp/backend/custom_kernels_mathematical.cpp

```diff
@@ -35,7 +35,7 @@ template <typename _KernelNameSpecialization>
 class custom_elemwise_absolute_c_kernel;
 
 template <typename _DataType>
-void custom_elemwise_absolute_c(void* array1_in, const std::vector<long>& input_shape, void* result1, size_t size)
+void custom_elemwise_absolute_c(void* array1_in, void* result1, size_t size)
```

Comment:
Why parameter `input_shape` is removed? What do we expect from the function (we do not need axis specific algorithm)?

Reply:
@shssf There is no axis param in function absolute.
https://numpy.org/doc/stable/reference/generated/numpy.absolute.html

Reply:
@shssf  absolute has not atribute `axis`.
https://numpy.org/doc/stable/reference/generated/numpy.absolute.html